Utilities/CommonDriver.cs

```diff
@@ -0,0 +1,14 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SpecProj2.Utilities
+{
+    public class CommonDriver
+    {
```

Comment:
finish implementing this

Reply:
please make use of Hooks , you might need to create the Common driver object there!
Please see
https://codebun.com/how-to-implement-hooks-in-specflow-c/